tests/jenkins/python/Util.py

```diff
@@ -37,25 +38,34 @@ def git_clone_repo(workdir, repo_name):
     else:
         url = 'https://github.com/UV-CDAT/' + repo_name
 
-    
-    cmd = 'git clone ' + url + ' ' + workdir + '/' + repo_name
-        
+    repo_dir = os.path.join(workdir, repo_name)
+    cmd = "git clone {url} {dest}".format(url=url, dest=repo_dir)
     ret_code = run_cmd(cmd)
     if ret_code != SUCCESS:
-        print("FAIL..." + cmd)
+        print("FAIL...{}".format(cmd))
         return ret_code
 
     return(ret_code)
 
 def run_in_conda_env(conda_path, env, cmds_list):
-    cmd = 'bash -c \"export PATH=' + conda_path + ':$PATH; '
-    cmd += 'source activate ' + env + '; '
-    
+
+    add_path_cmd = "export PATH={path}:$PATH".format(path=conda_path)
+    activate_cmd = "source activate {env}".format(env=env)
+    cmds = None
     for a_cmd in cmds_list:
-        cmd += a_cmd + '; '
-    cmd += 'source deactivate \"'
-    print('CMD: ' + cmd)
+        if cmds == None:
```

Comment:
The "correct" way to do this in python is
```
if cmds is None
```
[see here](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none). Shouldnt be an issue in this case, just something to keep in mind.